mongo_connector/oplog_manager.py

```diff
@@ -38,6 +38,46 @@
 LOG = logging.getLogger(__name__)
 
 
+class ReplicationLagLogger(threading.Thread):
+    """Thread that periodically logs the current replication lag.
+    """
+    def __init__(self, opman, interval=30):
+        super(ReplicationLagLogger, self).__init__()
+        self.opman = opman
+        self.interval = interval
+        self.daemon = True
```

Comment:
`daemon` was only [added as an argument to __init__ in Python 3.3](https://docs.python.org/3.5/library/threading.html#threading.Thread) so we should keep it as is for Python 2 compatibility.